examples/tutorials/device_asr.py

```diff
@@ -0,0 +1,166 @@
+"""
+Device ASR with Emformer RNN-T
+==============================
+
+**Author** : `Moto Hira <moto@fb.com>`__, `Jeff Hwang <jeffhwang@fb.com>`__.
+
+This tutorial shows how to use Emformer RNN-T and streaming API
+to perform device speech recognition.
```

Comment:
maybe something like the following would be clearer:

```suggestion
to perform speech recognition on a streaming device input.
```